tests/ga4gh/trs/test_server.py

```diff
@@ -1,11 +1,13 @@
 from flask import Flask
 from foca.models.config import Config, MongoConfig
 import mongomock
+import pytest
 
 from trs_filer.ga4gh.trs.server import (
     addTool,
+    toolsIdGet,
 )
-from trs_filer.errors.exceptions import exceptions  # noqa: F401
+from trs_filer.errors.exceptions import NotFound  # noqa: F401
```

Comment:
The reason why I imported `exceptions` though I don't use it in the file (which is why I put the `noqa: F401` pragma) is that I wanted to make sure that the `trs_filer/errors/exceptions.py` module is read by the interpreter (to achieve 100% coverage). As you are importing something from the module now for _another_ reason, the entire module is read anyway, and you can indeed omit the previous import. However, you _don't_ need to add the `noqa: F401` pragma, which stops `flake8` complaining about an unused import, because you _are_ indeed using `NotFound`. So please remove the pragma.